docs/eessi.md

```diff
@@ -0,0 +1,63 @@
+# EESSI Guide
+
+## How to load EESSI
+Loading an EESSI environment module:
```

Comment:
I think that:
```suggestion
Loading an EESSI environment module:
```
a) It needs to make clear these are alternatives, so something like "EESSI can be initialised using either ..."
b) the eessi module approach is marked as preferred in the docs, BUT there's the expandable section  "what if I already have Lmod installed ..." - which we do. So I think we need the bits from 1. in there too.